test/pdftotext.test.js

```diff
@@ -24,7 +24,7 @@ describe('pdftotext', function(){
     it('should throw exception', function(){
       var ptext = new pdftotext(__dirname + '/pdfs/invalidfile.pdf');
       function fn() {
-        var stdout = ptext.getTextSync();
+        return ptext.getTextSync().toString('utf8');
       }
       assert.throws(fn, /I\/O Error/);
```

Comment:
I will but I don't get why it's passing locally yet fails on the build.
node version is pretty similar.